fasthttprouter_tree.go

```diff
@@ -349,8 +349,10 @@ func (n *node) GetValue(reqPath string, ctx *Context, method string) (handle Req
 		panic("no cache!")
 	}
 	if record, ok := n.router.cache.Get(reqPath, method); ok {
-		for name, value := range record.values {
-			ctx.SetUserValue(name, value)
+		if ctx != nil {
+			for name, value := range record.values {
+				ctx.SetUserValue(name, value)
```

Comment:
If we'll omit this step, it may break named arguments

Reply:
Frankly, I am not familiar with the gramework as much as it could be at the moment, but I think it is ok and we could write a test for named arguments.